Utils/azuretools-core/src/com/microsoft/azuretools/telemetry/TelemetryConstants.java

```diff
@@ -0,0 +1,17 @@
+package com.microsoft.azuretools.telemetry;
+
+public class TelemetryConstants {
+    // production name
+    public static final String PRODUCTION_NAME_WEBAPP = "webapp";
+    public static final String PRODUCTION_NAME_ACR = "acr";
+    public static final String PRODUCTION_NAME_DOCKER = "docker";
+
+    // operation name
+    public static final String CREATE_WEB_APP = "create-webapp";
+    public static final String DELETE_WEB_APP = "delete-webapp";
+    public static final String DEPLOY_WEB_APP = "deploy-webapp";
+    public static final String DEPLOY_DEPLOYMENT_SLOT = "deploy-to-slot";
```

Comment:
the names "deploy-to-slot" and "create-deployment-slot" looks inconsistent.  maybe "deploy-deployment-slot"?

Reply:
deploy-deployment-slot seems too long, and the "deploy-to-slot" is just copy from the intellij plugin UI design

Reply:
For me, the prep. "to" in deploy-*to*-slot shouldn't be here as it provides no useful information. To debate, compare following names
```java
    public static final String DEPLOY_WEB_APP = "deploy-webapp";     // <-- why don't you name it "deploy-to-webapp"?
    public static final String DEPLOY_DEPLOYMENT_SLOT = "deploy-to-slot";
```

For simplicity, "deploy-slot" and "create-slot" look good to me.

Reply:
So just use deploy-webapp-slot, create-webapp-slot